### ISSUE TITLE
List of companies: which order?

### ISSUE THREAD

**Issue by pw374** [ocaml/v2.ocaml.org#345]
On #343 
- @agarwal said
  
  > We really should add some Javascript to randomize the order in which the list is shown on each refresh. Such code would come in handy in a few places. Or we can place all in alphabetical order, except randomly put a different one first (which is what used to be done on the old version of the site).
- @dbuenzli said
  
  > I think alphabetical order makes more sense when you want to refer to that page for telling someone/check for yourself that entity XXXX uses OCaml. Especially now that the list seems to be growing.

Meanwhile, I've made the list alphabetical.

> http://ocaml.org/learn/companies.html

**Comment by shayne-fletcher** [comment]
+1 (for alphabetical)

**Comment by amirmc** [comment]
Thanks for making a new issue. My general feeling is that it should be alphabetical but I also think it's important that 'household names' are prominent. 

Having companies like Facebook (and now Bloomberg) at/near the top is better from a casual browsing perspective. It demonstrates that OCaml is used by companies you've heard of.

**Comment by pw374** [comment]
If we don't have either a random list or an alphabetical list, I suggest we make 2 distinct lists, one with "known-to-everyone" companies and one with the other ones, the second being alphabetical.

**Comment by agarwal** [comment]
I agree. So let's just put Facebook and Bloomberg first, then the rest alphabetically. Not sure about Citrix; it is big, but not a _household_ name.

**Comment by avsm** [comment]
Well, Citrix isn't selling into the home. Close to anyone who works in a Fortune 500 company would have used a Citrix desktop remoting application at work...

**Comment by agarwal** [comment]
So you mean Citrix should go on the top too. Makes sense to me. They are known quite a lot, and perhaps even more so to ocaml.org's target audience, which is tech people.

**Comment by avsm** [comment]
I quite like how the fsharp.org page cites the sources.  There are a lot of CUFP talks on OCaml, so I wonder if we should start linking to those where relevant (definitely Facebook's, possibly my CUFP 2008 talk on Citrix/OCaml, and @yminsky's recent one on 10 years of Jane Street).

**Comment by agarwal** [comment]
I've put Facebook, Bloomberg, and Citrix on top. Remaining points are:
- Randomly showcase a different company first, on each page refresh. I find this less necessary now that we've showcased companies we think are the most important to give attention to.
- Link to videos relating to these companies. I've opened new issue #350 for this.

Unless someone thinks the first item is needed, we can close this issue.
